src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Colors {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
 export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea: () => number;
+}
+
+export function correctLengthChecker(...lengths: number[]): boolean {
+  return lengths.every((length: number) => length > 0);
+}
+
+export function isTrianglePossible(...sides: number[]): boolean {
+  sides.sort((sideA: number, sideB: number) => sideB - sideA);
 
+  return sides[0] < sides[1] + sides[2];
+}
+
+export function roundValueToHundrets(number: number): number {
```

Comment:
avoid such names, it is better to write universal functions
```suggestion
export function round(number: number, decimal: number): number {
```

`round(1.3423432, 2) => 1.34`
`round(1.3423432, 0) => 1`